combustion

```diff
@@ -100,15 +101,17 @@ if [ "${1-}" = "--prepare" ]; then
 			# Set rd.neednet if not already done and reevaluate it (module-specific)
 			getargbool 0 'rd.neednet' && exit 0
 			echo rd.neednet=1 > /etc/cmdline.d/40-combustion-neednet.conf
-			if [ -e /lib/dracut/hooks/pre-udev/60-net-genrules.sh ]; then
+			if [ -e "\$hookdir"/pre-udev/60-net-genrules.sh ]; then
```

Comment:
If the heredoc is converted to `<<"EOF"` or the sh call to a subshell this won't have to be escaped